public/test/setupTests.ts

```diff
@@ -1,12 +1,16 @@
 import '@testing-library/jest-dom';
 import { configure } from '@testing-library/react';
+import dotenv from 'dotenv';
 import i18next from 'i18next';
 import failOnConsole from 'jest-fail-on-console';
 import { initReactI18next } from 'react-i18next';
 
+dotenv.config();
+dotenv.config({ path: `.env.local`, override: true });
```

Comment:
I'm used to Vite handling it all for you, but here I had to load the main `.env` and the local one